dolly-backend-app/src/main/java/no/nav/dolly/bestilling/pdlforvalter/domain/PdlVegadresse.java

```diff
@@ -0,0 +1,25 @@
+package no.nav.dolly.bestilling.pdlforvalter.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PdlVegadresse {
+
+    private String adressekode;
+    private String adressenavn;
+    private String adressetillegsnavn;
```

Comment:
Du er jo bedre rettskriver enn meg, men skal det være adressetilleggsnavn med to g'er? 🤓

Reply:
Takk, men du har helt rett, skal være to gger.